src/tensor_api.cpp

```diff
@@ -199,6 +199,33 @@ extern "C" miopenStatus_t miopenSetTensorDescriptor(miopenTensorDescriptor_t ten
     });
 }
 
+extern "C" miopenStatus_t miopenSetTensorDescriptorEx(miopenTensorDescriptor_t tensorDesc,
+                                                      miopenDataType_t dataType,
+                                                      int nbDims,
+                                                      const std::size_t* dimsA,
+                                                      const std::size_t* stridesA)
```

Comment:
```suggestion
                                                      const size_t* dimsA,
                                                      const size_t* stridesA)
```

Reply:
I see mostly `std::size_t` in this file, but can change, if you insist (do you?)

Reply:
Fixed in 6dd37fb8f